### PR TITLE
TriggerEvent example

### PR DESCRIPTION
**Pull request by Poggicek**

### CODE REVIEW

File: content/docs/scripting-reference/runtimes/lua/functions/TriggerEvent.md [hunk 1]
```diff
@@ -20,4 +20,14 @@ TriggerEvent(string eventName[, ...])
...
 --------
 
-TODO
+Client
+```lua
+TriggerEvent('eventName', 'Hello world!')
+```
+Server
+```lua
+RegisterNetEvent('eventName')
```
Comment (Zeemahh) [inline content/docs/scripting-reference/runtimes/lua/functions/TriggerEvent.md:29]: You don't need to register as net event if you're not using it over the net, i.e. client->server/vice versa.

Comment (Poggicek) [reply content/docs/scripting-reference/runtimes/lua/functions/TriggerEvent.md:29]: Oh thats actually new for me. It does make sense

Comment (Poggicek) [reply content/docs/scripting-reference/runtimes/lua/functions/TriggerEvent.md]: fixed

**Comment by cerium-cfx** [comment]
`TriggerEvent` is for events within the same domain (e.g. client), so this example does not work. Both `AddEventHandler` and `TriggerEvent` must be on either the client or server. Please fix this

**Comment by technetium-cfx** [comment]
Please, test your examples. Triggering an event before it exists has no effect.

**Pull request closed without merging**